app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<main class="flex flex-col content-center items-center m-5 p-2">
+  <div class="header">
+    <h1 class="text-4xl font-extrabold dark:text-white m-5"><%= @recipe.name %></h1>  
+  </div>
+
+  <div class="food-container">
+  <div class="food-name">
+    <p>Preparation time: <%= @recipe.preparation_time %> minutes</p>
+    <p>Cooking time: <%= @recipe.cooking_time %> minutes</p>
+    <p><%= @recipe.description %></p>
+  </div>
+  <%= form_for @recipe, url: toggle_public_path(id: @recipe.id), method: :post do |f| %>
+    <div class="swith-btn">
+      <p><%= @recipe.public? ? 'Public' : 'Private' %></p>
+      <label class="switch">
+      <div><%= f.submit "" %></div>
+        <%= f.check_box :public, { checked: @recipe.public } %>
+        <span class="slider round"></span>
+      </label>
+    </div>
+    
+  <% end %>
+   
+  </div>  
+  <% if @recipe.user == current_user %>
+  <div class="flexbox">
+    <%= button_to 'Generate shopping List', shoppinglists_path(), method: :get, class: '' %>  
+    <%= button_to 'Add Ingredient', new_recipe_recipe_food_path(@recipe.id), method: :get, class: '' %>  
+  </div>
+  <% end %>
+
+<hr class="w-4/6 rounded-sm h-1 my-6 bg-gray-500" />
+
+<div class="center">
+<table class='w-3/6 mt-5 text-sm text-left text-gray-500 dark:text-gray-400'>
+  <thead class="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
+    <tr>
+      <th scope="col" class="px-6 py-3">Name</th>
+      <th scope="col" class="px-6 py-3">Quantity</th>
+      <th scope="col" class="px-6 py-3">Value</th>
+      <th scope="col" class="px-6 py-3">Actions</th>
+    </tr>
+  </thead>
+  <tbody>
```

Comment:
- [ ] when adding a food to a recipe with another quantity than 1 the application should calculate the total price for that food in the recipe, in your application it doesn't as you can see in below printscreen:
![image](https://user-images.githubusercontent.com/39240820/233590431-2f9d7ed0-5958-4cdf-a7f8-08e3ce2dae1e.png)

Reply:
Thank you very much 💯 . We have fixed it.